dev/lint-python

```diff
@@ -210,8 +210,8 @@ function black_test {
     local BLACK_STATUS=
 
     # Skip check if black is not installed.
-    $BLACK_BUILD 2> /dev/null
-    if [ $? -ne 0 ]; then
+    $PYTHON_EXECUTABLE -c 'import black' &> /dev/null
+    if [[ $? -ne 0 ]]; then
```

Comment:
Do we need this change? If so, we also need to modify https://github.com/apache/spark/pull/37305/files#diff-6f5ab2a3a5414ffdb0b3754c14a1b1f7427783101d86394a9fbdcf550970979cR27?

Reply:
I think we can leave single brackets, it's mostly me being used to it. I can revert if wanted.

Reply:
Shall we revert it then? At least could you make them consistent?

Reply:
Will revert for consistency.